src/components/PeopleManagement/data/hooks/useEnterpriseMembersTableData.js

```diff
@@ -4,6 +4,7 @@ import {
 import { camelCaseObject } from '@edx/frontend-platform/utils';
 import { logError } from '@edx/frontend-platform/logging';
 import debounce from 'lodash.debounce';
+import _ from 'lodash';
```

Comment:
nit: Can import `_.snakeCase` directly
```suggestion
import snakeCase from 'lodash/snakeCase';
```